test/logintest.js

```diff
@@ -0,0 +1,143 @@
+const request = require('supertest');
```

Comment:
please rename the file to login.test.js, so we know and can filter easily test files